### PR TITLE
fixed no-var warnings in test files

### PR DESCRIPTION
**Pull request by bizob2828**
## Proposed Release Notes
 * Removed warnings from agent tests for `no-var` eslint rule.

### CODE REVIEW

**Review by michaelgoin — APPROVED** [review]

File: test/integration/instrumentation/http.tap.js [hunk 1]
```diff
@@ -208,7 +208,7 @@ test('built-in http instrumentation should not swallow errors', function (t) {
     })
 
     // this is gonna blow up
-    // eslint-disable-next-line no-use-before-define
+    // eslint-disable-next-line no-use-before-define, no-var
     var x = x.dieshere.ohno
```
[the remark this answers is not in the document]
Comment (bizob2828) [reply test/integration/instrumentation/http.tap.js:212]: ah, i guess not. let me fix

File: test/integration/core/inspector.tap.js [hunk 2]
```diff
@@ -6,13 +6,10 @@
...
-  // eslint-disable-next-line node/no-unsupported-features/node-builtins
-  var inspector = require('inspector')
-} catch (e) {
-  // will skip tests below
-}
+// eslint-disable-next-line node/no-unsupported-features/node-builtins
+const inspector = require('inspector')
 const helper = require('../../lib/agent_helper')
+
 test('inspector', { skip: !inspector }, function (t) {
```
[the remark this answers is not in the document]
Comment (bizob2828) [reply test/integration/core/inspector.tap.js:13]: good catch!

File: test/benchmark/promises/shared.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ const tests = [
       const promises = [prom]
 
       for (let i = 0; i < internalPromises; ++i) {
-        var prom = promises[i]
+        prom = promises[i]
```
[the remark this answers is not in the document]
Comment (bizob2828) [reply test/benchmark/promises/shared.js:28]: kk, i'll dig a little more

**Comment by bizob2828** [comment]
ok @michaelgoin I think I address all your feedback

File: test/benchmark/promises/shared.js [hunk 4]
```diff
@@ -25,7 +25,6 @@ const tests = [
       const promises = [prom]
 
       for (let i = 0; i < internalPromises; ++i) {
-        prom = promises[i]
```
Comment (michaelgoin) [inline test/benchmark/promises/shared.js]: I think you still want this part as it is supposed to be chaining promises. My point was we don't need to reuse the outer 'prom'. The outer prom is just used to seed 'promises' and the inner could be scoped const just to the for loop. Its all not very critical but I think I unfortunately led you to changing what is being tested here. This setup is sort of awkward.

Comment (bizob2828) [reply test/benchmark/promises/shared.js]: lol, i see it now. my bad

**Review by garbados — APPROVED** [review]
LGTM!

**Review by michaelgoin — APPROVED** [review]

**Pull request merged**